lms/static/js/views/fields.js

```diff
@@ -20,12 +20,12 @@
             tagName: 'div',
 
             indicators: {
-                'canEdit': '<i class="icon fa fa-pencil message-can-edit" aria-hidden="true"></i>',
-                'error': '<i class="fa fa-exclamation-triangle message-error" aria-hidden="true"></i>',
-                'validationError': '<i class="fa fa-exclamation-triangle message-validation-error" aria-hidden="true"></i>',
-                'inProgress': '<i class="fa fa-spinner fa-pulse message-in-progress" aria-hidden="true"></i>',
-                'success': '<i class="fa fa-check message-success" aria-hidden="true"></i>',
-                'plus': '<i class="fa fa-plus placeholder" aria-hidden="true"></i>'
+                'canEdit': '<i class="icon fa fa-pencil message-can-edit" aria-hidden="true"><span class="sr">Editable</span></i>',
```

Comment:
@muhammad-ammar the SR text needs to be a sibling to the icon font.  right now it is a child of the icon font which has an aria-hidden="true" on it, which means it would never get exposed to the screen reader.  Try this:
```<i class="icon fa fa-pencil message-can-edit" aria-hidden="true"></i><span class="sr">Editable</span>```

apply to messages below: